test/hotspot/jtreg/runtime/ClassUnload/KeepAliveSoftReference.java

```diff
@@ -36,6 +36,8 @@
 import java.lang.ref.SoftReference;
 import jdk.test.whitebox.WhiteBox;
 import jdk.test.lib.classloader.ClassUnloadCommon;
+import java.util.List;
+import java.util.Set;
```

Comment:
This is unused in this test, but it may be better to check the returned set anyway rather than calling `wb.isClassAlive` at line 76.

Reply:
It's trying to match if the weak reference is cleared and comparing to isAlive so I thought that part of the test is still useful.   Only Set is unused so I removed that.